test/unit/puppetlabs/services/master/master_core_test.clj

```diff
@@ -11,6 +11,7 @@
         app         (build-ring-handler handler)
         request     (fn r ([path] (r :get path))
                           ([method path] (app (mock/request method path))))]
+    (is (= 200 (:status (request "/v3/environments"))))
```

Comment:
@nwolfe I thought about that, but then we'd have to special snowflake it since all the other routes have requests made to `/v3/path/foo', so I thought it would be better to just not add it to that sequence.